fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/fluid/FluidVariant.java

```diff
@@ -43,6 +50,12 @@
 @ApiStatus.Experimental
 @ApiStatus.NonExtendable
 public interface FluidVariant extends TransferVariant<Fluid> {
+	public static final Codec<FluidVariant> CODEC = RecordCodecBuilder.create(instance ->
```

Comment:
This should probably a `MapCodec` in case some people want to use it inline with other fields?